tests/unit/src/test/scala/tests/codeactions/OrganizeImportsLspSuite.scala

```diff
@@ -170,9 +187,61 @@ class OrganizeImportsLspSuite
        |  val a: Int = "no one wants unused imports"
        |}
        |""".stripMargin,
-    kind = List(kind),
+    kind = List(sourceKind),
     scalafixConf = scalafixConf(),
     scalacOptions = scalacOption,
     expectNoDiagnostics = false
   )
+
+  check(
+    "quickfix-available",
+    """
+      |import scala.util.Try<<>>
+      |
+      |object A {
+      |  val action = "quick should be available"
+      |}
+      |""".stripMargin,
+    s"${OrganizeImportsQuickFix.title}",
+    """
+      |
+      |
+      |object A {
+      |  val action = "quick should be available"
+      |}
+      |""".stripMargin,
+    kind = List(quickFixKind),
+    scalacOptions = scalacOption
+  )
+
+  check(
+    "quickfix-available-on-hover",
+    """
+      |import scala.@@util.Try
+      |
+      |object A {
+      |  <<val action = "quick should be available">>
+      |}
+      |""".stripMargin,
+    s"${OrganizeImportsQuickFix.title}",
```

Comment:
Turned out that the test is failing 😅. Is there a way to mimic hover behavior in tests?

Reply:
Quick fixes are unrelated to hovers, so no need to test them. I think the current test is `"on-unused"` is enough.

Reply:
Are you sure? In vscode when I hover unused there is quick fix action possible